queues/c-or-cpp/circular-queue-linked-list.cpp

```diff
@@ -0,0 +1,107 @@
+// Content: Circular Queue using linked list
+// Author: Tawfik Yasser
+// Date: Mon, 8 Mar 2021
+#include <iostream>
+using namespace std;
+class Node
+{
+public:
+    int data;
+    Node *next;
+    Node() {}
+};
+class QueueLL
+{
+private:
+    Node *front;
+    Node *rear;
+    int _length;
```

Comment:
although it's not a problem here but in future it is recommended to avoid naming identifiers starting with an underscore (`_`) as they are reserved for compiler implementation to use and using it might end up causing naming conflicts which might be difficult to diagnose as if that happens then the problem would be implantation dependent. 

it is also mentioned in C++ standards that : 
> Certain sets of names and function signatures are always reserved to the implementation:
>
> - Each name that contains a double underscore (__) or begins with an underscore followed by an uppercase letter (2.11) is reserved to the implementation for any use.
>
> - Each name that begins with an underscore is reserved to the implementation for use as a name in the global namespace.